modules/backend/layouts/_head.htm

```diff
@@ -56,7 +56,7 @@
         /* Only run on HTTPS connections
          * Block off Front-end Service Worker from running in the Backend allowing security injections, see GitHub #4384
          */
-        if (location.protocol === 'https:') {
+        if ((location.protocol === 'https:') && ('serviceWorker' in navigator)) {
```

Comment:
@ayumi-cloud thanks for the check now it should be good from my side!

Reply:
@w20k Good to have you reviewing this pr 👍